libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1266,6 +1271,10 @@ void CompilerUtils::rightShiftNumberOnStack(unsigned _bits)
 
 unsigned CompilerUtils::prepareMemoryStore(Type const& _type, bool _padToWords)
 {
+	solUnimplementedAssert(_type.canBeStored() && _type.isValueType(),
```

Comment:
should be `solAssert(_type.isValueType(), "This function only supports value types. Was used for " + _type.richIdentifier() + ".")`